tests/MongoDB.Driver.Core.Tests/Core/Servers/ServerTests.cs

```diff
@@ -480,11 +492,11 @@ internal void IsNotMaster_should_return_expected_result_for_message(string messa
         [InlineData((ServerErrorCode)(-1), false)]
         [InlineData(ServerErrorCode.NotMaster, true)]
         [InlineData(ServerErrorCode.InterruptedAtShutdown, true)]
-        internal void IsNotMasterOrRecovering_should_return_expected_result(ServerErrorCode code, bool expectedResult)
+        internal void IsStateChangeError_should_return_expected_result(ServerErrorCode code, bool expectedResult)
         {
             _subject.Initialize();
 
-            var result = _subject.IsNotMasterOrRecovering(code, null);
+            var result = _subject.IsStateChangeError(code, null);
```

Comment:
Renamed per the [spec](https://github.com/mongodb/specifications/commit/7dd9c008be6edaf65368787d5e91c2ac3aeb95b8#diff-62b4da2e38e7d2a9c1a2d92d315895bbR1368).